benchmark/streams/writable-manywrites.js

```diff
@@ -34,7 +34,7 @@ function main({ n, sync, writev, callback }) {
 
   let k = 0;
   function run() {
-    while (k++ < n && s.write(b, cb));
+    while (k++ < n && s.write(b, 'buffer', cb));
```

Comment:
What about adding an `encoding` option? I guess not passing through any encoding is the common case, so keeping that would be good. To reduce the overall runtime `n` could probably be lowered to 1e6 at the same time.